extensions/amp-ad-exit/0.1/amp-ad-exit.js

```diff
@@ -136,13 +151,23 @@ export class AmpAdExit extends AMP.BaseElement {
             const vendorResponse = replacements./*OK*/expandStringSync(
                 customVar.iframeTransportSignal, {
                   'IFRAME_TRANSPORT_SIGNAL': (vendor, responseKey) => {
+                    if (!(vendor && responseKey)) {
+                      return '';
+                    }
                     const vendorResponses = this.vendorResponses_[vendor];
                     if (vendorResponses && responseKey in vendorResponses) {
                       return vendorResponses[responseKey];
                     }
                   },
                 });
-            if (vendorResponse != '') {
+            if (customVar.iframeTransportSignal ==
+                `IFRAME_TRANSPORT_SIGNAL${vendorResponse}`) {
+              // No substitution occurred, so format string in amp-ad-exit
+              // config was invalid
+              dev().error(TAG, 'Invalid IFRAME_TRANSPORT_SIGNAL format:' +
+                  vendorResponse +
+                  ' (perhaps there is a space after a comma?)');
```

Comment:
This refers to line 58 of extensions/amp-ad-network-fake-impl/0.1/data/fake_amp_ad_with_iframe_transport.html - "a,b" is valid but "a, b" is not.

Reply:
Why isn't the space valid?

Reply:
I'm not sure. I would guess the split() call at https://github.com/ampproject/amphtml/blob/master/src/service/url-replacements-impl.js#L920 should be split(/,\s*/) instead of split(',') but that's just from a quick look. I would be happy to investigate more but would prefer to break that into a different PR.

Reply:
That, and we need to add a space in https://github.com/ampproject/amphtml/blob/master/src/service/variable-source.js#L244. Please file a feature request.

```diff
# This could be reworked anyways
- new RegExp('\\$?(' + all + ')(?:\\(([0-9a-zA-Z-_.,]+)\\))?', 'g');
+ new RegExp('\\$?(' + all + ')(?:\\(([0-9a-zA-Z-_., ]+)\\))?', 'g');
```

Reply:
Created https://github.com/ampproject/amphtml/issues/11616

Reply:
I guess that's to prevent space in URL?